pandemania/src/api/Toasts.js

```diff
@@ -0,0 +1,13 @@
+import { toast } from 'react-toastify';
+
+export const apiErrorToast = () => {
+	toast.error('Covid API id down, please try again later', { toastId: 'api-error' });
+};
+
+export const loginSuccessToast = () => {
+	toast.success('Successfully logged in!', { toastId: 'login' });
```

Comment:
Consider ending every toast with punctuation mark.

Reply:
added "!" to every toast